.gitignore

```diff
@@ -0,0 +1 @@
+/target/
```

Comment:
I generally like to use https://www.gitignore.io/api/java%2Clinux%2Cmacos%2Cwindows%2Ceclipse%2Cintellij

Reply:
My goodness that's a lot of content.  I recommend against using that.  Prefer to include a minimal set of _project_-specific ignores to avoid accidentally ignoring things that should go into your repo.